lib/tasks/release-notes.js

```diff
@@ -0,0 +1,21 @@
+module.exports = async (ctx, task) => {
+    const got = require('got');
+    let response;
+
+    try {
+        response = await got('https://api.github.com/repos/TryGhost/Ghost/releases', {json: true});
```

Comment:
Was going to mention this before it got merged, but I think we should have a timeout here (between 500ms and 1.5s) since by default there isn't one

Reply:
that's a good point, yeah